README.md

```diff
@@ -1,73 +1,32 @@
 # Metals
 
+[![Join the chat at https://gitter.im/scalameta/scalameta](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scalameta/scalameta)
 [![](https://travis-ci.org/scalameta/metals.svg?branch=master)](https://travis-ci.org/scalameta/metals)
 
-This project is an experiment to implement a [Language Server](https://github.com/Microsoft/language-server-protocol)
-for Scala using Scalameta semanticdb and the Scala presentation compiler.
+This project is an experiment to implement a
+[Language Server](https://github.com/Microsoft/language-server-protocol) for
+Scala using [Scalameta](http://scalameta.org/)-related technologies.
 
-:warning: This project is very alpha stage.
-Expect bugs and surprising behavior.
-Ticket reports and patches are welcome!
+:warning: This project is under development and is not intended to be used for
+day-to-day coding. Expect bugs and incomplete documentation. Installation
+instructions are primarily intended for project contributors.
 
 ## Project Goals
 
-This project has the following goals:
-
-- a good UX for the final users, including simple installation and setup
-- low memory requirements
-- integration with scalameta-based tools, such as [Scalafix](https://github.com/scalacenter/scalafix) and [Scalafmt](https://github.com/scalameta/scalafmt)
-
-## Roadmap
```

Comment:
I'm OK with moving away the roadmap, but I think readme should at least link to it.

Reply:
Done